lstchain/high_level/hdu_table.py

```diff
@@ -323,30 +311,38 @@ def set_expected_pos_to_reco_altaz(data):
     """
     Set expected source positions to reconstructed alt, az positions for
```

Comment:
This formulation is a bit misleading. Seems to state the opposite (i.e. take the reco values and put them in the "expected" variables) that what is actually done.
Be more specific, e.g. "Fill the reconstructed alt az with the expected source position. This is just a trick..."

Reply:
Fixed the docstring. Maybe we can update the function name as well, to something like `use_expected_pos_for_reco_altaz` or `fill_reco_altaz_w_expected_pos`. As this function is only used for source-dependent analysis @SeiyaNozaki do you have a preference?

Reply:
I think the function name is fine